### PR TITLE
Feature/code structuring

### PR DESCRIPTION
**Pull request by velisav-boyanov**
Organized the main method in game.py as @shristov-elsys wanted.

### CODE REVIEW

File: game.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	if input == "/draw/" and draw_requested != id:
+		draw_requested = id
+		return handle_draw()
+	elif input == "/draw/" and draw_requested == id:
+		return ["You send the offer, you can't accept it!"]
+
+	if input == "/cancel_draw/":
+		cancle_draw()
+		return ["Draw wasn't accepted"]	
+
```
Comment (nikitaprodanov) [inline game.py:61]: I see you have used too many "-if" statements. They are too cpu-consuming and expensive. Could you consider using a "switch case:" or hashMap?

**Review by nikitaprodanov — CHANGES_REQUESTED** [review]
Everything looks good except for the multiple "if" statements in game_input() method.

File: game.py (hunk 1 above)
Comment (velisav-boyanov) [reply game.py:61]: Pyhton doesn't have switches and even if I use dictionaries or hash maps I'll still have to use if statements.

Comment (nikitaprodanov) [reply game.py:61]: That's my bad. You area right seems like the "if...elif...else" sequence in python is optimized enough.

**Review by nikitaprodanov — APPROVED** [review]

**Pull request merged**